### DISCUSSION TITLE
Problem with OLED display

### DISCUSSION THREAD

**Discussion by luk3ros** [sanni/cartreader#737] [Need help]
Hello,
I have just build Card Reader Version 3 few minutes ago. I have problem with screen that displays on OLED. It looks like not all horizontal lines are drawn. The bottom of the screen is not affected. Everything is fine there. I have shown it in the photo. I resoldered the connections from both sides of the OLED PCB again, but it didn't change anything. Is the screen damaged and needs to be replaced? Please help. 
![Sanni Cart reader](https://user-images.githubusercontent.com/119793366/222981289-2fe4395a-33d8-433c-bf0e-90dfc7820bc8.jpg)

**Comment by sanni** [comment]
I had a case like this once, buying a new OLED fixed it.

**Comment by luk3ros** [comment]
Thank you for your answer. I ordered a new screen today. I'll let you know if the problem is solved. Regards.

**Reply by luk3ros** [reply]
Hello,
Just want to say that new screen soveld problem :)
![1 (2)](https://user-images.githubusercontent.com/119793366/227745457-d1ec5673-d1ca-4387-b078-355c6cac3266.jpg)
Thank You :D
